peachjam/admin.py

```diff
@@ -362,6 +400,13 @@ class IngestorAdmin(admin.ModelAdmin):
     inlines = [IngestorSettingInline]
     readonly_fields = ("last_refreshed_at",)
     form = IngestorForm
+    actions = ["reset_ingestor_refresh_date"]
+
+    def reset_ingestor_refresh_date(self, request, queryset):
```

Comment:
@actlikewill I think it would be useful if this also queued up the background task  to refresh this ingestor.